### PR TITLE
fix: fix some selectors

### PR DESCRIPTION
**Pull request by Jingyu-Yang-Qytera**

### CODE REVIEW

File: .idea/.gitignore
Comment (csvtuda) [inline .idea/.gitignore:1]: Please do not add this file. The documentation should not be a "proper" software project.

File: .idea/Qytera-Gmbh.github.io.iml
Comment (csvtuda) [inline .idea/Qytera-Gmbh.github.io.iml:1]: Please do not add this file. The documentation should not be a "proper" software project.

File: .idea/modules.xml
Comment (csvtuda) [inline .idea/modules.xml:1]: Please do not add this file. The documentation should not be a "proper" software project.

File: .idea/vcs.xml
Comment (csvtuda) [inline .idea/vcs.xml:1]: Please do not add this file. The documentation should not be a "proper" software project.

**Review by csvtuda — CHANGES_REQUESTED** [review]

File: .idea/.gitignore
Comment (Jingyu-Yang-Qytera) [reply .idea/.gitignore:1]: Habe ich gelöscht.

File: .idea/Qytera-Gmbh.github.io.iml
Comment (Jingyu-Yang-Qytera) [reply .idea/Qytera-Gmbh.github.io.iml:1]: Habe ich gelöscht.

**Pull request merged**